src/main/java/life/qbic/presenter/tabs/workflows/WorkflowTypePresenter.java

```diff
@@ -0,0 +1,98 @@
+package life.qbic.presenter.tabs.workflows;
+
+import com.vaadin.ui.Accordion;
+import com.vaadin.ui.VerticalLayout;
+import life.qbic.model.view.GridModel;
+import life.qbic.presenter.MainPresenter;
+import life.qbic.presenter.tabs.ATabPresenter;
+import life.qbic.view.TabView;
+import life.qbic.view.tabs.GridView;
+import submodule.lexica.ChartNames;
+import submodule.lexica.Translator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author fhanssen
+ */
+public class WorkflowTypePresenter extends ATabPresenter<GridModel, GridView> {
+
+    private final String workflowType;
+    private final List<String> subTypes;
+    private final List<AvailableWorkflowsPresenter> availableWorkflowsPresenters;
+
+    private final Accordion accordion = new Accordion();
+
+    public WorkflowTypePresenter(MainPresenter mainPresenter, String workflowType) {
+        super(mainPresenter, new GridView(1, 2, true, true));
+
+        this.workflowType = workflowType;
+        this.subTypes = new ArrayList<>();
+        this.availableWorkflowsPresenters = new ArrayList<>();
+
+        extractData();
+        addChartSettings();
+        addChartData();
+
+    }
+
+    @Override
+    public void extractData() {
+        String subType = Translator.getOriginal(workflowType);
+        if (subType.isEmpty()) {
+            subType = workflowType;
+
+        }
+
+        final String s = subType;
+        getMainPresenter().getMainConfig().getCharts().keySet().forEach(n -> {
+            if (n.contains(ChartNames.Available_Workflows_.toString()) && n.contains(s)) {
+                subTypes.add(n);
+            }
+        });
+
+    }
+
+    @Override
+    public void addChartSettings() {
+        super.setModel(new GridModel("title", "subtitle"));
+        logger.info("Settings were added to " + this.getClass() + " with chart title: " + super.getModel().getTitle());
+
+    }
+
+    @Override
+    public void addChartData() {
+        subTypes.forEach(s -> {
+            AvailableWorkflowsPresenter ap = new AvailableWorkflowsPresenter(getMainPresenter(), s);
+            availableWorkflowsPresenters.add(ap);
+        });
```

Comment:
In your other presenter classes you're logging addingChartData as well and for consistency's sake I suggest that you log here as well.

Reply:
good point, will add it before merging